server/jwt_test.go

```diff
@@ -5048,8 +5049,9 @@ func TestJWScopedSigningKeys(t *testing.T) {
 			}
 		}
 	}
-	errHdlr := nats.DisconnectErrHandler(func(conn *nats.Conn, err error) {
+	errHdlr := nats.ErrorHandler(func(conn *nats.Conn, s *nats.Subscription, err error) {
 		errChan <- err
+		t.Log(err)
```

Comment:
Actually we should not log things with `t.` (unless when you are debugging). This is adding noise to the test run logfile.

Reply:
fixed